mpsl/fem/CMakeLists.txt

```diff
@@ -4,7 +4,17 @@
 #
 
 function(mpsl_fem_append_lib fem_lib_name)
-    set(LIB_DIR "${CMAKE_CURRENT_SOURCE_DIR}/${fem_lib_name}/lib/${CONFIG_MPSL_LIB_DIR}/${CONFIG_MPSL_LIB_FLOAT_ABI_DIR}")
+    if (CONFIG_SOC_SERIES_NRF54HX OR CONFIG_SOC_SERIES_NRF54LX)
```

Comment:
@hermabe Seems this change got merged in this nrfxlib fyi

Reply:
Then this will break the upmerge which will be merged soon(tm).

Reply:
I will create a new update with the backport included